tests/libmonitoring/prometheus_utils.go

```diff
@@ -1,4 +1,4 @@
-package monitoring
+package libmonitoring
```

Comment:
I would remove the `utils` suffix  from the file name.

Reply:
Done